doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_ironside.rst

```diff
@@ -0,0 +1,694 @@
+* Boot commands (ERASEALL, DEBUGWAIT)
+* CPUCONF service
+* Update service
+* PSA crypto service
+
+.. _ug_nrf54h20_ironside_se_uicr:
+
+Global Resource configuration
+*****************************
+
+|ISE| uses the User Information Configuration Registers (UICR) to securely configure persistent global resource settings.
+The UICR is a dedicated non-volatile memory region (and its associated registers) located in the first NVR page (NVR0).
+It holds device-specific configuration data, such as application-owned memory boundaries and selected peripheral parameters, which are used to configure the system before the application starts.
+
+.. note::
+   When you configure a custom devicetree for your board, the build system automatically generates UICR content based on the devicetree configuration.
+
+The following UICR fields are supported:
+
++----------------------+---------------------------------------------------------------------+
+| UICR Field           | Description                                                         |
++======================+=====================================================================+
+| UICR.VERSION         | A 32-bit value that encodes the UICR format version (16-bit major   |
+|                      | and 16-bit minor).                                                  |
++----------------------+---------------------------------------------------------------------+
+| UICR.LOCK            | Locks all contents of NVR0, preventing any further writes without   |
+|                      | performing an ERASEALL operation.                                   |
++----------------------+---------------------------------------------------------------------+
+| UICR.APPROTECT       | Configures debugger and access-port permissions for each AP via the |
+|                      | TAMPC peripheral.                                                   |
++----------------------+---------------------------------------------------------------------+
+| UICR.ERASEPROTECT    | Blocks ERASEALL commands to NVR0.                                   |
++----------------------+---------------------------------------------------------------------+
+| UICR.PROTECTEDMEM    | Defines the size (in 4 KiB blocks) of an integrity-checked memory   |
+|                      | region at the start of the application-owned part of MRAM.          |
++----------------------+---------------------------------------------------------------------+
+| UICR.RECOVERY        | Holds application-specific parameters used by the recovery          |
+|                      | mechanism.                                                          |
++----------------------+---------------------------------------------------------------------+
+| UICR.ITS             | Specifies the base address and size of Internal Trusted Storage     |
+|                      | for application and radio domains.                                  |
```

Comment:
These two can maybe be deleted for now. They are not supported by the firmware yet, and will look slightly different in the next version.